pgtap/ksp/ksp/types_check.pg

```diff
@@ -18,26 +18,34 @@ along with this program; if not, write to the Free Software
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  ********************************************************************PGR-GNU*/
 BEGIN;
-/* SELECT plan(5); */
-SELECT plan(3);
+SELECT plan(5); 
 
 SELECT has_function('pgr_ksp');
 SELECT has_function('pgr_ksp',ARRAY['text','bigint','bigint','integer','boolean','boolean']);
 SELECT function_returns('pgr_ksp',ARRAY['text','bigint','bigint','integer','boolean','boolean'],'setof record');
 
-/* TODO
 
 SELECT set_eq(
   $$SELECT  proargnames from pg_proc WHERE proname = 'pgr_ksp'$$,
-  $$SELECT  '{"","","","","directed","heap_paths","seq","path_id","path_seq","node","edge","cost","agg_cost"}'::TEXT[] $$
+  $$VALUES
+  ('{"","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[])
```

Comment:
I think the records are duplicated. No use of duplicating the records when using set_eq - https://pgtap.org/documentation.html#set_eq